ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/searchresults/v1/searchresults/paginationbar.html

```diff
@@ -0,0 +1,101 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~
+  ~ Copyright 2019 Adobe. All rights reserved.
+  ~ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License. You may obtain a copy
+  ~ of the License at http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software distributed under
+  ~ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  ~ OF ANY KIND, either express or implied. See the License for the specific language
+  ~ governing permissions and limitations under the License.
+  ~
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<template data-sly-template.pagination="${@ searchResultsSet}">
+	<div class="category__pagination">
+		<div class="category__pagination_root">
+			<sly data-sly-list.page="${searchResultsSet.paginationParameters}">
+				<!--/*First Page button & Previous page button inactive */-->
+				<sly data-sly-test="${pageList.first && searchResultsSet.searchOptions.currentPage==1} ">
+					<a class=" category__pagination_navbutton-inactive category__pagination_navbutton-arrow category__pagination_tilebutton" aria-label="move to the first page">
+                        <span class="category__pagination_icon-root">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="1.2rem" height="1.2rem" viewBox="0 0 24 24" fill="#999" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><polygon points="11 19 2 12 11 5 11 19"></polygon><polygon points="22 19 13 12 22 5 22 19"></polygon></svg>
```

Comment:
We should extract these in CSS

Reply:
I think this makes a LOT of sense and would be ideal, but this pattern seems to be used in a number of places in the repo so I hesitate to add yet another new convention in this already massive PR. That said, is there a place this has already been accomplished that you're aware of (replacing the svg with a CSS rule for these particular elements?)